docs/manual/common/guide/devmode/CassandraServer.md

```diff
@@ -156,4 +156,25 @@ In sbt:
 
 @[local-instance](code/build-cassandra-opts3.sbt)
 
-Assuming your local Cassandra instance is running on port `9042`.
+These two settings will only be used when running Lagom in DevMode. The purpose of these two settings is to disabled the embedded Cassandra server and configure the Service Locator in DevMode to still be able to locate Cassandra when looking for `cas_native`.
+
+The service locator setup in these examples assumes your local Cassandra instance is running on port `9042`.
+
+## Disabling Service Location
```

Comment:
I'm not sure that this section really belongs in the page on the development mode Cassandra server. Maybe it should be on the page about configuring Cassandra persistent entities?